analytical/templatetags/intercom.py

```diff
@@ -8,8 +8,8 @@
 import hmac
 import json
 import sys
-import time
 import re
+import time
```

Comment:
Is there a specific reason why you (or our editor) is changing the import order? Aren't they all standard library modules?

Reply:
Alphabetic order, I guess...